docs/source/index.rst

```diff
@@ -10,7 +10,27 @@
 JupyterLab Documentation
 ########################
 
-JupyterLab is the next-generation web-based user interface for Project Jupyter. `Try it on Binder <https://mybinder.org/v2/gh/jupyterlab/jupyterlab-demo/3818244?urlpath=lab/tree/demo>`__.  JupyterLab follows the Jupyter `Community Guides <https://jupyter.readthedocs.io/en/latest/community/content-community.html>`__.
+Welcome to the **JupyterLab** documentation site. JupyterLab is a highly
+extensible, feature-rich notebook authoring application and editing environment,
+and is a part of `Project Jupyter <https://docs.jupyter.org/en/latest/>`_, a
+large umbrella project centered around the goal of providing tools (and `standards <https://docs.jupyter.org/en/latest/#sub-project-documentation>`_) for interactive
+computing with `computational notebooks <https://docs.jupyter.org/en/latest/#what-is-a-notebook>`_.
```

Comment:
Note: this depends on https://github.com/jupyter/jupyter/pull/682 which adds such a section.

Reply:
(I am implying we - i.e. someone with merge rights - should merge the other PR first),